lib/tasks/school_groups.rake

```diff
@@ -0,0 +1,16 @@
+namespace :school_groups do
+  desc 'Create default school groups and scoreboards'
+  task create_default_school_groups_and_scoreboards: :environment do
```

Comment:
@jamesjefferies I've just added a rake task for populating some example data. The after_party gem doesn't look like it's been updated to support Rails 5 (although, I think it _might_ work on Rails 5) so I'm wary of introducing it as a dependency https://github.com/theSteveMitchell/after_party

Reply:
@jamesalmond ok - well, delving in to producing a rails5 compliant version maybe a simple PR for after_party or it may be a flippin' nightmare - so rake task for now I guess